src/components/SettingsView.js

```diff
@@ -86,37 +85,43 @@ export default class SettingsView extends Component {
     return (
 
       <View style={styles.main}>
-        {header}
```

Comment:
I know code in this file is really ugly. I've already fixed it, it will be added in the next pull request.